tests/signature_tests/test_ipsignature.py

```diff
@@ -0,0 +1,34 @@
+import unittest
+from scapy.all import rdpcap
+from scapy.layers.inet import IP
+from src.signature_detection.ip_signature import IPSignature
+from pathlib import Path
+
+
+class TestIPSignature(unittest.TestCase):
+    def setUp(self) -> None:
+        self.ip_signature = IPSignature("192.168.1.0/24")
+        packets = rdpcap(str(Path("../tests/test_data/test_tcpdump2.pcap").resolve()))
```

Comment:
Is this pcap too big to add in the PR?

Reply:
It's not in the actual merge, to run these tests you can create a pcap with this name locally

Reply:
But the pcap has to be a specific file so the first two are malicious and the second two are benign. If it's small enough can you add it so all the tests are working off the same file?